pkgs/development/python-modules/pywick/default.nix

```diff
@@ -0,0 +1,45 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytest
+, h5py
+, hickle
+, numpy
+, pandas
+, pillow
+, six
+, pytorch
+, torchvision
+, tqdm
+, lib
+}:
+
+buildPythonPackage rec {
+  pname   = "pywick";
+  version = "0.5.6";
+
```

Comment:
```suggestion
```

Reply:
I find this a bit cluttered, but feel free to ping again if it's become the convention or something.

Reply:
i think there was 2 newline between version and src. The current diff looks good

Reply:
i was looking at the other package.

Generally disabled is immediately below, I consider this related to the version information usually, so it makes sense for me to have it adjacent to it. But since you're the maintainer, I'm fine with some flexibility